Code/EnginePlugins/AngelScriptPlugin/TypeRegistration/AsRegister_Reflected.cpp

```diff
@@ -395,10 +388,16 @@ void ezAngelScriptEngineSingleton::RegisterGenericFunction(const char* szTypeNam
 
 void ezAngelScriptEngineSingleton::RegisterSingleGenericFunction(const char* szFuncName, const char* szTypeName, const ezAbstractFunctionProperty* const pFunc, const ezScriptableFunctionAttribute* pFuncAttr, bool bIsInherited, const ezRTTI* pReturnType)
 {
-  ezStringBuilder decl;
   bool bVarArgs = false;
+  if (const ezDynamicPinAttribute* pVarArgsAttr = pFunc->GetAttributeByType<ezDynamicPinAttribute>())
```

Comment:
Is this ezDynamicPinAttribute something that we can rely on? Also should we maybe assert, that it is set for vararg params?